iphone/Classes/TiUIClipboardProxy.m

```diff
@@ -339,49 +328,58 @@ - (void)setItems:(id)args
       YES);
 }
 
-- (id)getItems:(id)unused
+- (NSArray<NSDictionary<NSString *, id> *> *)getItems
 {
-  __block id items;
-
+  __block NSMutableArray<NSDictionary<NSString *, id> *> *result = [[[NSMutableArray alloc] init] retain];
   TiThreadPerformOnMainThread(
       ^{
-        items = [[[self pasteboard] items] retain];
+        NSArray<NSDictionary<NSString *, id> *> *items = [self pasteboard].items;
 
         // Check for invalid UTI's / mime-types to prevent a runtime-crash
-        for (NSDictionary *item in items) {
-          for (NSString *key in [item allKeys]) {
-            if ([key hasPrefix:@"dyn."]) {
```

Comment:
Does this check not required?

Reply:
I didn't like this check because it "hid" the underlying problems we had sticking data into and pull it back out of the clipboard. If it's an unknown data type, it would get this `'dyn.'` prefix. And that meant if anything in the clipboard had it, we'd basically not return anything. I'll add it back to skip `'dyn.'` entries only (but still turn valid/known ones)